llvm/test/Transforms/ConstraintElimination/unreachable-bb.ll

```diff
@@ -0,0 +1,36 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -p constraint-elimination -S %s | FileCheck %s
+
+define void @f(i32 noundef %v0, i32 noundef %v1, i32 noundef %v2) {
```

Comment:
could you add variants of the test with different permutations of the AND's and add a version with OR's as well?